spec/unit/resources/splunk_conf_spec.rb

```diff
@@ -4,8 +4,8 @@
 include CernerSplunk::ConfHelpers, CernerSplunk::ResourceHelpers
 
 shared_examples 'splunk_conf' do |platform, version, package|
-  describe 'action :configure' do
-    let(:runner_params) { { platform: platform, version: version, user: 'root' } }
+  let(:runner_params) { { platform: platform, version: version, user: 'root' } }
+  chef_context 'action :configure' do
```

Comment:
I feel like contextually this doesn't work. In rspec, although `describe` and `context` are functionally the same, contextually they are differentiated in that `describe` tests against one functionality while the `context` is testing functionality against a given state.

I know you wrote the chef_context to get around the weirdness around contexts, but we might need a similar chef_describe if we want to be contextually clear.